operators/hack/deployer/runner/settings.go

```diff
@@ -57,3 +63,24 @@ type RunConfig struct {
 	Id        string                 `yaml:"id"`
 	Overrides map[string]interface{} `yaml:"overrides"`
 }
+
+func ParseFiles(plansFile, runConfigFile string) (plans Plans, runConfig RunConfig, err error){
+	var yml []byte
+	yml, err = ioutil.ReadFile(plansFile)
+	if err == nil {
+		err = yaml.Unmarshal(yml, &plans)
+		if err != nil {
+			return
+		}
+	}
+
+	yml, err = ioutil.ReadFile(runConfigFile)
```

Comment:
I find the flow with named return values a bit harder to parse. Also are we not hiding at least one I/O error if both `ReadFile(plansFile)` and `ReadFile(runConfigFile)` fail?

Reply:
Ah I missed @barkbay 's comment, which goes into the same direction.